openmp/runtime/CMakeLists.txt

```diff
@@ -347,10 +347,6 @@ if(LIBOMP_OMPD_SUPPORT AND ((NOT LIBOMP_OMPT_SUPPORT) OR (NOT "${CMAKE_SYSTEM_NA
   set(LIBOMP_OMPD_SUPPORT FALSE)
 endif()
 
-# OMPX Taskgraph support
```

Comment:
I'm not sure if I asked it before, but if possible can we run some task benchmark suites, like BOLT, to see the overhead? If no overhead, it'd be great to remove the macro.

Reply:
Sure, do you know where can we find the BOLT benchmark suite? Thanks!